dwave_networkx/drawing/chimera_layout.py

```diff
@@ -51,6 +51,13 @@ def chimera_layout(G, scale=1., center=None, dim=2):
         Number of dimensions. When dim > 2, all extra dimensions are
         set to 0.
 
+    plot_kwargs : None or dict (default None)
+        A dict of keyword arguments to be used in a plotting function (see
+        :func:`networkx.draw` and :func:`draw_lineplot`), to scale node and edge
+        sizes appropriately to the graph `G`.  Optional keys in ``plot_kwargs``
+        are set to default values by the :func:`normalize_size_and_aspect`
```

Comment:
":func:`normalize_size_and_aspect`" --> two issues:
1. Needs to be something like :func:`~dwave_networkx.drawing.qubit_layout.normalize_size_and_aspect`
2. You can only link to this function once you add it to an RST file so it's rendered to the docs.